packages/manager/apps/web-domains/src/domain/components/ContactCards/HolderCards.tsx

```diff
@@ -0,0 +1,114 @@
+import { useTranslation } from 'react-i18next';
+import { NAMESPACES } from '@ovh-ux/manager-common-translations';
+import { useNavigationGetUrl } from '@ovh-ux/manager-react-shell-client';
+import { useNavigate } from 'react-router-dom';
+import {
+  useGetDomainContact,
+  useGetDomainResource,
+} from '@/domain/hooks/data/query';
+import { TDomainContact } from '@/common/types/common.types';
+import { LegalFormEnum } from '@/common/enum/common.enum';
+import {
+  Button,
+  BUTTON_SIZE,
+  Card,
+  Spinner,
+  SPINNER_SIZE,
+  TEXT_PRESET,
+  Text,
+  MessageBody,
+  Message,
+} from '@ovhcloud/ods-react';
+
+interface HolderCardsProps {
+  readonly serviceName: string;
+  readonly isAdminConnected: boolean;
+}
+
+export default function HolderCards({
+  serviceName,
+  isAdminConnected,
+}: HolderCardsProps) {
+  const { t } = useTranslation(['domain']);
+  const navigate = useNavigate();
+  const { domainResource } = useGetDomainResource(serviceName);
+
+  const contactID =
+    domainResource.currentState.contactsConfiguration.contactOwner.id;
+
+  let editContactUrl: string = '';
+  let domainContact: TDomainContact = null;
+
+  if (isAdminConnected) {
+    const { domainContact: dc } = useGetDomainContact(contactID);
+    domainContact = dc;
+
+    const { data: url } = useNavigationGetUrl([
+      'web',
+      `/domain/${serviceName}/contact-management/edit-contact/${contactID}/`,
+      {},
+    ]);
+    editContactUrl = url as string;
+
+    if (!domainContact) {
+      return <Spinner size={SPINNER_SIZE.xs} />;
+    }
+  }
+
+  return (
+    <Card
+      data-testid="holder-card"
+      className="flex w-full flex-col h-full p-6 justify-between"
+    >
+      <div className="flex flex-col gap-y-6 mb-6 text-[--ods-color-heading]">
+        <Text preset={TEXT_PRESET.heading4}>
```

Comment:
```suggestion
        {isAdminConnected ? 
        <Text preset={TEXT_PRESET.heading4}>
         {t('domain_tab_contact_management_holder_title')}
         {`(${contactID})`}
        </Text>
        :
        <Message dismissible={false} className="mb-6 w-full">
            <MessageBody>{...}
```
If you use the component text, even with no value, it may add a space to your HMI